python/lsst/daf/butler/core/datasets.py

```diff
@@ -96,51 +96,50 @@ class DatasetRef(object):
     ----------
     datasetType : `DatasetType`
         The `DatasetType` for this `Dataset`.
-    units : `dict`
+    dataId : `dict`
```

Comment:
That is because it is wrong. In the new design `DataUnit`s will be visible as `dict`s (because the instances of `DataUnit` represent tables, not rows therein).